src/main/java/org/broadinstitute/consent/http/service/DaaService.java

```diff
@@ -102,6 +109,33 @@ public DataAccessAgreement findById(Integer daaId) {
     throw new NotFoundException("Could not find DAA with the provided ID: " + daaId);
   }
 
+  public void sendDaaRequestEmails(User user, Integer daaId) throws Exception {
+    try {
+      List<Institution> institutions = institutionService.findAllInstitutions();
+      Institution institution = institutions.stream()
+            .filter(inst -> inst.getId().equals(user.getInstitutionId()))
+          .findFirst()
+          .orElseThrow();
+      List<SimplifiedUser> signingOfficials = institution.getSigningOfficials();
+      if (signingOfficials.isEmpty()) {
+        throw new NotFoundException("No signing officials found for user: " + user.getDisplayName());
+      }
+      int userId = user.getUserId();
+      String userName = user.getDisplayName();
+      for (SimplifiedUser signingOfficial : signingOfficials) {
+        String signingOfficialName = signingOfficial.displayName;
+        String signingOfficialEmail = signingOfficial.email;
```

Comment:
It's up to you, but since these are referenced only once and are descriptive without being too verbose, I'd pass them directly to the method.

Reply:
These two variables are reset upon iteration through all the signing officials for an institution, so I'm not sure if I would be able to pass them into the method unless I'm missing something.

Reply:
https://github.com/DataBiosphere/consent/pull/2283/files/ab23cd2eab405263105196c5aafd2c789b7ab2af#r1559889542 here is a concrete suggestion!

Reply:
Ah, I completely misread your suggestion. That makes sense, thank you for the clarification!